integreat_cms/cms/templates/pois/poi_form_sidebar/related_contacts_box.html

```diff
@@ -50,7 +50,9 @@
                  class="bg-green-100 border-l-4 border-green-500 text-green-800 px-4 py-3 hidden">
                 {% trans "The new contact was successfully created." %}
             </div>
-            <div id="contact-ajax-error-message"
+            <div id="contact-ajax-error-message">
+            </div>
+            <div id="contact-ajax-unexpected-error-message"
                  class="bg-red-100 border-l-4 border-red-500 text-red-700 px-4 py-3 hidden">
```

Comment:
With multiple error messages able to pop up simultaneously, we should put some minimal space between them so they don't appear as one visual object

```suggestion
                 class="bg-red-100 border-l-4 border-red-500 text-red-700 px-4 py-3 my-1 hidden">
```